src/main/java/com/github/princesslana/eriscasper/util/SendableImage.java

```diff
@@ -0,0 +1,25 @@
+package com.github.princesslana.eriscasper.util;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+import javax.imageio.ImageIO;
+
+public abstract class SendableImage {
+
+  BufferedImage image;
+
+  @Override
+  public String toString() {
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    try {
+      ImageIO.write(image, "jpg", baos);
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Don't output to stdout. Does it make sense to return empty string rather than throw an exception here?

Reply:
we should probably throw an exception, it doesn't make sense to have an invalid image anywhere at all, especially sending it to discord.

Reply:
jpq is more common than png, I'll look for a way to accept both but that's the way I found to move a image into Base64, I'll look int png and gif options in a few.

Reply:
jpg doesn't support transparency. Only gif supports animation. Forcing to a single image type isn't great behavior here.

Reply:
Yeah I'm adding in implementation for other format types.